### PR TITLE
docs: update docs for fhevmjs

### PR DESCRIPTION
**Pull request by immortal-tofu**

### CODE REVIEW

File: docs/guides/webapp.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ import { initFhevm, createInstance } from "fhevmjs";
 const init = async () => {
   await initFhevm(); // Load TFHE
   return createInstance({
+    kmsContractAddress: "0x208De73316E44722e16f6dDFF40881A3e4F86104",
+    aclContractAddress: "0xc9990FEfE0c27D31D0C2aa36196b085c0c4d456c",
     network: window.ethereum,
```
Comment (jatZama) [inline docs/guides/webapp.md:71]: change this `network`to `networkUrl`

File: docs/guides/webpack.md [hunk 2]
```diff
@@ -48,8 +48,15 @@ If you have an issue with bundling the library (for example with some SSR framew
...
 const start = async () => {
   await window.fhevm.initFhevm(); // load wasm needed
-  const instance = window.fhevm.createInstance({ chainId, publicKey }).then((instance) => {
-    console.log(instance);
-  });
+  const instance = window.fhevm
+    .createInstance({
+      kmsContractAddress: "0x208De73316E44722e16f6dDFF40881A3e4F86104",
+      aclContractAddress: "0xc9990FEfE0c27D31D0C2aa36196b085c0c4d456c",
+      network: window.ethereum,
```
Comment (jatZama) [inline docs/guides/webpack.md:55]: here also change `network` to `networkUrl`

File: docs/references/fhevmjs.md [hunk 3]
```diff
@@ -47,6 +51,8 @@ Using `window.ethereum` object:
 import { createInstance } from "fhevmjs";
 
 const instance = await createInstance({
+  kmsContractAddress: "0x208De73316E44722e16f6dDFF40881A3e4F86104",
+  aclContractAddress: "0xc9990FEfE0c27D31D0C2aa36196b085c0c4d456c",
   network: window.ethereum,
```
Comment (jatZama) [inline docs/references/fhevmjs.md:56]: here also, 3rd place to change

**Review by jatZama — CHANGES_REQUESTED** [review]

File: docs/references/fhevmjs.md (hunk 3 above)
Comment (immortal-tofu) [reply docs/references/fhevmjs.md:56]: `network` is the parameter if you send a EIP1193 object, as written in the doc
```
- `networkUrl` or `network`: the URL or Eip1193 object provided by `window.ethereum` - used to fetch chainId and KMS nodes' public key
```

Comment (jatZama) [reply docs/references/fhevmjs.md:56]: ok sorry for the confusion

**Review by jatZama — APPROVED** [review]

**Pull request merged**